libraries/AP_Math/crc.cpp

```diff
@@ -282,3 +282,28 @@ void hash_fnv_1a(uint32_t len, const uint8_t* buf, uint64_t* hash)
         *hash *= FNV_1_PRIME_64;
     }
 }
+
+/*
+  calculate 24 bit RTCMv3 crc. We take an approach that saves memory
```

Comment:
Is this specific to RTCMv3?

If so, it shouldn't be moved.

If not... comment needs updating.

Reply:
It is currently used only on the rtcmv3, in this form, I didn't dig if we use this crv elsewhere.
But I think the change is good as it regroup the crc in the right file for easier maintenance and it save some flash so that is  enough to me as improvement.
I missed the comment, that need to be cleaned